app/initializers/register-showdown-extensions.js

```diff
@@ -2,12 +2,13 @@ import showdown from 'showdown';
 
 export function initialize() {
   showdown.extension('youtubeEmbed', () => {
-    const youtubeRegex = /.*(\$\(youtu.be\/|v\/|e\/|u\/\w+\/|embed\/|v=)([\w-]+)\).*/g;
```

Comment:
The `youtu.be` part can be removed here, because in the new regex the `youtu.` domain is matched and the `e\/` matches the end of the `.be` domain

Reply:
Could you maybe extract the regex and include some tests? To prevent this kind of regression